### PR TITLE
KK-29 | Add graphql to the project

### PR DESCRIPTION
**Pull request by badrange**
- add script that fetches typings from schema (    `"graphql-types": "apollo client:codegen --target=typescript --globalTypesFile='src/__generated__/globalTypes.ts'"`
)

### CODE REVIEW

**Comment by badrange** [comment]
- add script that fetches typings from schema (    "graphql-types": "apollo client:codegen --target=typescript --globalTypesFile='src/__generated__/globalTypes.ts'"
)
-

File: apollo.config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  client: {
+    service: {
+      name: 'kukkuu',
+      url: 'http://localhost:8001/graphql',
```
Comment (midori0507) [inline apollo.config.js]: config.js ? -> .ts instead?

File: package.json [hunk 2]
```diff
@@ -73,6 +79,7 @@
       "src/**/*.{ts,tsx}",
       "!<rootDir>/node_modules/",
       "!src/index.tsx",
+      "!src/domain/graphql/generated",
```
Comment (midori0507) [inline package.json]: Make `src/domain/api` instead of `graphql` name

File: src/domain/graphql/generated/globalTypes.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/* tslint:disable */
+/* eslint-disable */
+// This file was automatically generated and should not be edited.
+
+//==============================================================
```
Comment (midori0507) [inline src/domain/graphql/generated/globalTypes.ts]: if folder name can change from `generated` to `generatedTypes` would be nice

File: src/domain/auth/userManager.ts [hunk 4]
```diff
@@ -8,6 +8,7 @@ const location = `${window.location.protocol}//${window.location.hostname}${
 /* eslint-disable @typescript-eslint/camelcase */
 const settings: UserManagerSettings = {
   authority: process.env.REACT_APP_OIDC_AUTHORITY,
+  automaticSilentRenew: true,
```
Comment (midori0507) [inline src/domain/auth/userManager.ts]: nope

File: src/domain/home/form/HomePreliminaryForm.tsx [hunk 5]
```diff
@@ -32,6 +33,7 @@ interface HomeFormValues {
 
 interface Props {
   setFormValues: (values: RegistrationFormValues) => void;
+  submittedFormValues: RegistrationFormValues;
```
Comment (midori0507) [inline src/domain/home/form/HomePreliminaryForm.tsx]: Why you connect the data but you dont use it, should be removed

File: apollo.config.js (hunk 1 above)
Comment (badrange) [reply apollo.config.js]: Required adding `export {}` to the file to address ts complaint: 

> Cannot compile namespaces when the '--isolatedModules' flag is provided

File: apollo.config.ts [hunk 6]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
```
Comment (midori0507) [inline apollo.config.ts]: use ES6 export
`
export default {content}
`

File: package.json (hunk 2 above)
Comment (midori0507) [inline package.json]: This has not been updated with new path

**Comment by midori0507** [comment]
@badrange You might want to include script in README doc? not gonna hurt anybody

File: package.json (hunk 2 above)
Comment (badrange) [reply package.json]: Oo! Good spot!

File: apollo.config.ts (hunk 6 above)
Comment (badrange) [reply apollo.config.ts]: That doesn't work with the apollo command. Changing back to js file

**Pull request merged**